plugins/Actions/Columns/ActionType.php

```diff
@@ -40,12 +32,45 @@ class ActionType extends ActionDimension
 
     public function __construct()
     {
-        $this->acceptValues = sprintf('A type of action, such as: %s', implode(', ', $this->types));
+        $this->acceptValues = sprintf('A type of action, such as: %s', implode(', ', $this->getEnumColumnValues()));
```

Comment:
Can we maybe hard code some of the values here instead of listing them all? The text says "such as" anyway. The thing is that we otherwise on each request request all actions etc which makes page load time a wee bit slower than it has to be. We can still execute it, just saying ideally we would only list a few to not make things even slower.